src/includes/getting-started-next-steps/javascript.angular.mdx

```diff
@@ -0,0 +1,11 @@
+- **[Upload Source Maps](/platforms/javascript/sourcemaps/)**
```

Comment:
this file looks the same as the javascript file?

Reply:
My error - I should have removed this when I added <PageGrid header="Next Steps" nextSteps />. Thanks for the catch.